CONTRIBUTING.md

```diff
@@ -1,9 +1,8 @@
 # Contributing
 
-The core developer team of Fleek Non Fungible Apps welcomes all contributions from the community to achieve consistent code updates and enhancements.
-
-If you want to help us improve the project, you can always start a conversation through the [issues page](https://github.com/fleekxyz/non-fungible-apps/issues). Make sure you are being respectful towards the members of the community. We take our open-source community seriously and hold ourselves and other contributors to high communication standards.
+The core developer team of Fleek Non-Fungible Apps welcomes all contributions from the community to achieve consistent code updates and enhancements.
 
+If you want to help us improve the project you can always start a conversation through the [issues page](https://github.com/fleekxyz/non-fungible-apps/issues). Please, make sure you are respectful towards all members of the community. We take our open-source community seriously and hold ourselves and our contributors to high communication standards.

```

Comment:
I would say start a conversation through the discussions section or if you spot an issue then create an issue.